src/resolvers-cassandra/Messages/queries.js

```diff
@@ -51,8 +93,8 @@ function byLocation(args, res) { // eslint-disable-line no-unused-vars
     AND externalsourceid = ?
     AND centroidlat = ?
     AND centroidlon = ?
-    AND eventtime >= ?
-    AND eventtime <= ?
+    AND eventime >= ?
```

Comment:
Good catch, I made this change to enable testing against the cluster but it should be reverted.